packages/mobile/android/app/build.gradle

```diff
@@ -198,7 +198,7 @@ android {
         testBuildType System.getProperty('testBuildType', 'debug')
         testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
         ndk {
-            abiFilters "arm64-v8a"
+            abiFilters "arm64-v8a", "x86_64", "x86"
```

Comment:
We intentionally target only arm64 devices (there're only 33 x86 and x86_64 devices)

<img width="559" alt="Screenshot 2024-04-25 at 09 46 34" src="https://github.com/TryQuiet/quiet/assets/15381135/f77f345e-bd87-43bf-b607-f43f170fdce8">

Reply:
I don't think it's a good idea to bring it back because we lack x86 binaries for Tor, NodeJS and classic-level anyways

Reply:
Okay, yes you're right. I misunderstood what the abi is, and misattributed what made my build start succeeding. I'll close this PR.